shaq/_cli.py

```diff
@@ -186,6 +190,12 @@ def main() -> None:
         else:
             print(f"Track: {track.track.title}")
             print(f"Artist: {track.track.subtitle}")
-
+            if args.albumcover:
+                album_cover = raw["track"]["images"]["coverart"]
+                album_cover_hq = album_cover.replace(
+                    "/400x400cc.jpg", "/1000x1000cc.png"
+                )
+                print(f"Album Cover: {album_cover_hq}")
```

Comment:
Making sure I understand: this is taking a URL from the raw JSON response, and replacing the 400x400 cover filename with a 1000x1000 filename? Does Shazam guarantee that the higher-resolution cover exists?

Reply:
The URL is Apple's image servers. Most covers that I've seen are larger than 1000x1000.
Apple Music's minimum resolution is [1400x1400](https://itunespartner.apple.com/music/support/5215-digital-packaging-music#:~:text=The%20recommended%20file%20size%20is,PNG%20(100%25%20quality). "Digital Packaging for music"), so it may be more sensible to set it to that?
If there is no higher-res cover, the URL will still display the largest resolution available from the image servers.

Reply:
No strong preference from me -- is there a doc that guarantees that the largest available is always served from these URLs? If so, using 1400x1400 is fine and IMO we should add a comment explaining what's happening here.

Reply:
Can't find any docs.
The only evidence I have is that if you set the URL to an insanely high resolution, it will force the server to fetch the largest album cover image available.
For example, if you [set the URL to 10000x10000, it returns with a 1440x1440 image.](https://is1-ssl.mzstatic.com/image/thumb/Music/e7/c2/c2/mzi.dyvibegj.jpg/10000x10000cc.png)

Reply:
Cool -- in that case, could you paraphrase that into the comment? No link is fine as long as we explain what's happening here, to minimize surprise to a future editor 🙂